### DISCUSSION TITLE
Using PythonNet on Linux to embed python code in c#

### DISCUSSION THREAD

**Discussion by AghilesD** [pythonnet/pythonnet#1855] [Q&A]
### Environment

-   Pythonnet version: 
-   Python version: 3.6
-   Operating System: Linux (Ubuntu)
-   .NET Runtime: Microsoft.AspNetCore.App 6.0.6

### Details

-   Describe what you were trying to get done.

==> I am trying to use python code in my C# program. For this, I tried to do it with Pythonnet.

-   What commands did you run to trigger this issue? If you can provide a [Minimal, Complete, and Verifiable example (http://stackoverflow.com/help/mcve) this will help us understand the issue.

    **Instalation** 

       To install Pythonnet i  used  "pip3 install git+https://github.com/pythonnet/pythonnet"

       I had : 

          Processing /home/greenerwave/pythonnet
          Installing collected packages: UNKNOWN
          Running setup.py install for UNKNOWN ... done
          Successfully installed UNKNOWN-0.0.0

      The problem is that when i execute " pythonnet --version" i have " pythonnet : command not found"**

      So, I don't know if pythonnet is really installed ???

      **Use of pythonnet with C# in visual studio code** 

      I 'am new user of c#, so i dont now how to import pythonnet in my c# project with visual studio code : 

      Should I include someThing inside me .csproj ?
      Should I specify something else?  

![image](https://user-images.githubusercontent.com/108793776/177514467-2c59a1b8-71b0-4df3-9053-39e3c5893a7c.png)

Could you please help me with these two topics?

**Comment by Minhhnh** [comment]
Did you fix it, I also having this problem right now and there is no guide for this.
